templates/base.lds

```diff
@@ -259,7 +259,7 @@ SECTIONS
     PROVIDE( metal_segment_data_target_start = ADDR(.data) );
     PROVIDE( metal_segment_data_target_end = ADDR(.tdata) + SIZEOF(.tdata) );
 
-    .tbss : {
+    .tbss : ALIGN(8) {
```

Comment:
I think we need ALIGN(8) on the .tbss_space section as well, to make sure the space allocated for tbss is the same as the amount we used.

Reply:
I've made another commit to align .tbss_space as well.  What about .ctors, .dtors, .text, .tdata, and .rodata?

Reply:
As others have mentioned, those *should* all get automatically aligned by the linker.

Reply:
We also need to align .tdata -- if .data doesn't end on an aligned address, then all of the TLS data will end up unaligned, and that's bad...